TaskManagerTLA.DAL/Repositories/Interfaces/IAssignedTaskRepo.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using TaskManagerTLA.DAL.Entities;
+
+namespace TaskManagerTLA.DAL.Repositories.Interfaces
+{
+    public interface IAssignedTaskRepo<T, I> : IRepository<T, I>
```

Comment:
Немає толку з IAssignedTaskRepo<T, I>, T I всеодно ніде в тебе не використовуються нижче
```suggestion
    public interface IAssignedTaskRepo : IRepository<AssignedTask, string>
```